### PR TITLE
Resolve "Create custom ThemeProvider"

### PR DESCRIPTION
**Pull request by lcsrinaldi**

### CODE REVIEW

File: src/utils/font-faces.js [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+        url('https://fonts.picter.com/charter/83fadcc0-5f33-4dfc-af40-cac829e5be93.eot?#iefix') format('eot'),
+        url('https://fonts.picter.com/charter/ddf06504-e618-494d-b1b9-56e26c0bf664.woff2') format('woff2'),
+        url('https://fonts.picter.com/charter/291df853-a088-4f20-9bb4-ce8097a4011b.woff') format('woff'),
+        url('https://fonts.picter.com/charter/3f4f26ae-3bd3-4b3c-91e0-daf64c948eda.ttf') format('truetype'),
+        url('https://fonts.picter.com/charter/279fffe9-9054-428f-be89-3c9cb4f57c2d.svg#279fffe9-9054-428f-be89-3c9cb4f57c2d') format('svg');
+    }
+
+    #root {
+      overflow: auto;
+      height: 100%;
```
Comment (noxan) [inline src/utils/font-faces.js]: Could be confusing to have these styles within some `injectHostedFonts` function or?

File: src/utils/font-faces.js [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+    @font-face {
+      font-family: 'ITC-Charter';
+      src: url('https://fonts.picter.com/charter/bd4a6c95-033c-41f0-9510-2de2be19ebc6.eot?#iefix');
+      src:
+        url('https://fonts.picter.com/charter/bd4a6c95-033c-41f0-9510-2de2be19ebc6.eot?#iefix') format('eot'),
+        url('https://fonts.picter.com/charter/6b354187-add6-421c-9022-918efeb1a9d4.woff2') format('woff2'),
+        url('https://fonts.picter.com/charter/fcf607e0-9726-46de-99a0-3d80440fcad0.woff') format('woff'),
+        url('https://fonts.picter.com/charter/3a077c2d-60d8-4c1e-947f-64cd5c57b382.ttf') format('truetype'),
+        url('https://fonts.picter.com/charter/da20fd94-cdae-454e-a5d9-e1790eeaf24b.svg#da20fd94-cdae-454e-a5d9-e1790eeaf24b') format('svg');
+    }
```
Comment (noxan) [inline src/utils/font-faces.js]: Can't we load the with the same syntax as used for `Calibre`? It looks way more structured.

File: src/utils/font-faces.js [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+  }),
+};
+
+export const injectHostedFonts = () =>
+  injectGlobal`
+    ${calibreWeb.light};
+    ${calibreWeb.regular};
+    ${calibreWeb.medium};
+
+    @import url('https://fast.fonts.net/t/1.css?apiType=css&projectid=9e55868e-ed78-4479-9384-180f50fc2b61');
```
Comment (noxan) [inline src/utils/font-faces.js]: We should check if it is fine to release this tracking code to the public. But it is included in our web apps anyhow... do you know more on this @mattii?

File: src/utils/font-faces.js (hunk 2 above)
Comment (lcsrinaldi) [reply src/utils/font-faces.js]: I tried to do that, but the utility function doesn't support different URLs for different formats.

https://polished.js.org/docs/#fontface

File: src/utils/font-faces.js (hunk 1 above)
Comment (lcsrinaldi) [reply src/utils/font-faces.js]: Sorry, included by mistake.

File: src/utils/font-faces.js (hunk 3 above)
Comment (mattii) [reply src/utils/font-faces.js]: Yes, it's a problem, if we offer Prisma as ready to use open source library. We secured the font-files with CORS settings, so they're restricted to picter-domains. But this tracking code is mapped to our account at fonts.com and tracks page-views.

**Pull request merged**